src/Integration/LocalServices/IProjectSystemHelper.cs

```diff
@@ -97,6 +97,13 @@ internal interface IProjectSystemHelper : ILocalService
         string GetProjectProperty(Project dteProject, string propertyName);
 
         /// <summary>
+        /// Get the value of a given MSBuild project property for the specified build configuration (e.g. "Debug")
+        /// </summary>
+        /// <param name="propertyName">Name of the property to get</param>
+        /// <returns>The value of the property or null if the property does not exist/has not been set.</returns>
+        string GetProjectProperty(Project dteProject, string propertyName, string configuration);
```

Comment:
Added an override so the _configuration_ can be specified since we now need to ask for the _CodeAnalysisRuleSetDirectories_ property for a specific configuration.